test/error-placement.js

```diff
@@ -291,6 +291,42 @@ test( "test existing non-label used as error element", function(assert) {
 	assert.noErrorFor( field );
 });
 
+test( "test aria-describedby with input names contains special characters (:, ., [, ], $)", function() {
```

Comment:
are there even more selector-meta-characters which have this problem? e.g. `(`, `)`